src/lib/ical/utils.ts

```diff
@@ -150,10 +150,9 @@ export function produceSummary(title: string, description: string, selfLink: str
     : title + ' — ' + selfLink;
 }
 
-export function replaceHtmlWithExternalLinks(html: string): string {
-  return html.replace(/<a .*href=".*".*>/gm, (match: string): string => {
-    if (match.includes('target="_blank"')) return match;
-    match = match.replace(/target=".*?"\W*/g, '');
+export function replaceHtmlLinkTargets(html: string, withTarget = '_blank'): string {
+  return html.replace(/<a\W.*?href=".*?".*?>/gm, (match: string): string => {
+    match = match.replace(/target=".*?"\W*/gm, '');
     return match.slice(0, match.length - 1) + ' target="_blank">';
```

Comment:
To apply a custom target (or `"_blank"` by default) we can try embedding the `withTarget` argument like so,

```suggestion
    return match.slice(0, match.length - 1) + ` target="${withTarget}">`;
```